### ISSUE TITLE
Cannot run tests on non-default host

### ISSUE THREAD

**Issue by ikudjoi** [snowflakedb/snowflake-connector-net#77]
### Issue description
The connector tests are trying to run the tests agains server [account].snowflakecomputing.com. There is no way to configure the tests to use other server instead, e.g. [account].eu-west-1.snowflakecomputing.com instead.

### Example code
Run tests with parameters.json using server that is not located at US west AWS region.

### Error log
Running tests from command line will result in this error message.
```
Error Message:
 System.AggregateException : One or more errors occurred. (A task was canceled.)
  ----> System.Threading.Tasks.TaskCanceledException : A task was canceled.
```

### Configuration
*Driver version:*
Snowflake.Data 1.0.4

*Dotnet framework and version:*
.net standard 2.0

*Server version:*
3.0.5

*Client OS:* 
Windows 10

**Comment by ikudjoi** [comment]
I'm about to place a pull request to fix this.

**Comment by howryu** [comment]
Merged #78
